docs/en/integrations/data-ingestion/clickpipes/kafka.md

```diff
@@ -104,45 +104,29 @@ The supported formats are:
 - [JSON](../../../interfaces/formats.md/#json)
 - [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent)
 
-### JSON
 
+### Supported Data Types
 
-#### Supported data types
+The following ClickHouse data types are currently supported in ClickPipes:
 
-The following ClickHouse types are currently supported for JSON payloads:
-
-- Base numeric types
-  - Int8
-  - Int16
-  - Int32
-  - Int64
-  - UInt8
-  - UInt16
-  - UInt32
-  - UInt64
-  - Float32
-  - Float64
+- Base numeric types - \[U\]Int8/16/32/64 and Float32/64
+- Large integer types - \[U\]Int128/256
+- Decimal Types
 - Boolean
 - String
 - FixedString
 - Date, Date32
-- DateTime, DateTime64
+- DateTime, DateTime64 (UTC timezones only)
 - Enum8/Enum16
-- LowCardinality(String)
+- UUID
+- IPv4
+- IPv6
+- all ClickHouse LowCardinality types
 - Map with keys and values using any of the above types (including Nullables)
 - Tuple and Array with elements using any of the above types (including Nullables, one level depth only)
-- JSON/Object('json'). experimental
-
-:::note
-Nullable versions of the above are also supported with these exceptions:
-
-- Nullable Enums are **not** supported
-- LowCardinality(Nullable(String)) is **not** supported
```

Comment:
Do we support `LowCardinality(Nullable(String))` now? otherwise, this callout might still be helpful since we say we support "all ClickHouse LowCardinality types"